brigade-worker/src/app.ts

```diff
@@ -1,11 +1,11 @@
 /**
  * Module app is the main application runner.
  */
-import * as events from "./events";
+import * as events from "@azure/brigadier/out/events";
```

Comment:
nit: I am mildly annoyed by the fact that `/out/` should be included every time the package is used - I suspect there's a configuration property that can help us avoid this - I'll look into it tomorrow morning.

Reply:
Ok, mildly was an understatement.

Reply:
Update: found a solution for pretty import paths - not ideal, but will PR against Brigadier.